### PR TITLE
Jormungandr: use geocodejson/reverse API to get an object from coords

### PR DESCRIPTION
**Pull request by TeXitoi**

### CODE REVIEW

File: source/jormungandr/tests/bragi_autocomplete_tests.py [hunk 1]
```diff
@@ -485,7 +485,9 @@ def test_journey_with_external_uri_from_bragi(self):
             'https://host_of_bragi/autocomplete?q=bobette&{p}'.format(p=params): (bragi_bobette, 200),
             'https://host_of_bragi/features/bobette?pt_dataset=main_routing_test': (bragi_bobette, 200),
             'https://host_of_bragi/autocomplete?q=20+rue+bob&{p}'.format(p=params): (bob_street, 200),
-            'https://host_of_bragi/features/{}?pt_dataset=main_routing_test'.format(check_utils.r_coord): (bob_street, 200)
+            'https://host_of_bragi/reverse?lat={lat}&lon={lon}&pt_dataset=main_routing_test'
```
Comment (antoine-de) [inline source/jormungandr/tests/bragi_autocomplete_tests.py:488]: we don't care about the pt_dataset no ?

**Review by antoine-de — APPROVED** [review]

**Review by pbougue — APPROVED** [review]

File: source/jormungandr/tests/bragi_autocomplete_tests.py (hunk 1 above)
Comment (TeXitoi) [reply source/jormungandr/tests/bragi_autocomplete_tests.py:488]: Maybe that will be the same interface for place nearby, giving a limit and some types. In this case, we care.

I can remove it if you want, or just let it like that by default as it's less work without identified problems.

**Comment by antoine-de** [comment]
Jenkins, test this please

**Comment by antoine-de** [comment]
Jenkins, test this please

**Comment by antoine-de** [comment]
Jenkins, test this please

**Comment by xlqian** [comment]
Jenkins, test this please!

**Pull request merged**